config-generators/cosmosdb_nosql-commands.txt

```diff
@@ -2,13 +2,14 @@ init --config "dab-config.CosmosDb_NoSql.json" --database-type "cosmosdb_nosql"
 add Planet --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.planet" --permissions "anonymous:create,read,update,delete" --graphql "Planet:Planets"
 update Planet --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.include "*"
 update Planet --config "dab-config.CosmosDb_NoSql.json" --permissions "authenticated:create,read,update,delete"
-add Character --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.character" --permissions "anonymous:create,read,update,delete" --graphql "Character:Characters"
-add StarAlias --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.star" --permissions "anonymous:create,read,update,delete" --graphql "Star:Stars"
-update StarAlias --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.star" --permissions "authenticated:create,read,update,delete"
-add TagAlias --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.tag" --permissions "anonymous:create,read,update,delete" --graphql "Tag:Tags"
-add Moon --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.moon" --permissions "anonymous:create,read,update,delete" --graphql true
-update Moon --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.moon" --permissions "authenticated:create,read,update,delete"
-add Earth --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.earth" --permissions "field-mutation-with-read-permission:read" --graphql "Earth:Earths"
+update Planet --config "dab-config.CosmosDb_NoSql.json" --permissions "limited-read-role:read"
+add Character --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.planet" --permissions "anonymous:create,read,update,delete" --graphql "Character:Characters"
```

Comment:
as per our discussion, the entity level source.object value of the container should be taking precedence. So, we should rather fix the test at https://github.com/Azure/data-api-builder/blob/fa419cd46fe12d16633ce9bb317140c342d38183/src/Service.Tests/CosmosTests/TestBase.cs#L135 to keep separate container and check the functionality as long as there's no cross container join

Reply:
This can be covered as separate PR, I am noting down all these scenarios in the same sheet to include them later.

Reply:
Sure, you can open a tracking issue as well.

Reply:
@sourabh1007 please open GitHub issues to track

Reply:
already included in my tracking sheet.